bot.user.js

```diff
@@ -579,7 +579,8 @@ window.computeFoodGoal = function() {
 			for (var j = 0; j < nIter; ++j) {
 				var p2 = window.sortedFood[j];
 				var dist = window.getDistance(p1.xx, p1.yy, p2.xx, p2.yy);
-				if (dist < 100) {
+				
+				if (dist < getSnakeWidth() * 5) {
```

Comment:
Too big. it should be aiming at clusters it can completely eat, no? (i.e. one snake width)

I'm going to try both 5x and 1x to see what works better.